include/ofi_mr.h

```diff
@@ -56,6 +56,7 @@ struct ofi_mr_info {
 	struct iovec iov;
 	enum fi_hmem_iface iface;
 	uint64_t device;
+	uint64_t id;
```

Comment:
Please add a blank line above id and rename id to peer_id.  (The blank line is to help identify the field as an ipc related value.)  Please rename other places where you added id to peer_id, because without the commit message, what id actually is isn't obvious.